modular_skyrat/code/modules/mob/living/carbon/human/species_types/speciessnowflake.dm

```diff
@@ -0,0 +1,95 @@
+
+/datum/species/skeleton
+	screamsounds = list('modular_citadel/sound/voice/scream_skeleton.ogg')
+	fluff_desc = "Skeletons are simply put the result of failed experimentation and magic. Not much is know about the process to become one."
+
+/datum/species/fly
+	screamsounds = list('modular_citadel/sound/voice/scream_moth.ogg')
+	fluff_desc = "Flypeople are most commonly transmuted people, who suffered extreme changes to their biology because of botched teletransportation technology."
+
+/datum/species/insect
+	screamsounds = list('modular_citadel/sound/voice/scream_moth.ogg')
+	fluff_desc = "Insect people come from varying sectors of the galaxy, and their background varies wildly."
+
+/datum/species/insect/moth
+	screamsounds = list('modular_citadel/sound/voice/scream_moth.ogg')
+	fluff_desc = "Moth people seem to be a splinter of insect people species, and are very similar in biology and appearance."
+
+/datum/species/human
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Humans come from the Sol system, and are the commanding species of both NanoTrasen and SolGov."
+
+/datum/species/angel
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Not much is know about angels, but they are most commonly associated with christian mythology."
+
+/datum/species/corporate
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Corporate robots are simply advanced corporate androids employed by NT. Most of the time, they are high ranking CC officers."
+
+/datum/species/dullahan
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Not much is known about the dullahan, but they sure know how to make someone lose their minds."
+
+/datum/species/dwarf
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Dwarves are very similar to humans in biology, but they seem to have completely split from them as a species."
+
+/datum/species/mammal
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Anthropomorphs encompass a lot of the know sapient species by SolGov, and their backgrounds always vary wildly."
+
+/datum/species/jelly
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Slimepeople are the result of scientifical experimentation on slimes, which NanoTrasen has heavily invested in. At the current day, most slimepeople are simply consumers of brands of slime genemods sold by NanoTrasen."
```

Comment:
Slimepeople are by and large the results of experimentation on various slime entities in control of various organizations, chief of which being Nanotrasen and their Xenobiological Research Department. While a vast majority of Slimepeople are the result of this experimentation, there exists many gel-like alien species in galaxy, along with those resultant from Nanotrasen brand Slime-Hybrid genemods.